Cargo.toml

```diff
@@ -11,7 +11,7 @@ edition = "2018"
 [dependencies]
 arrayvec = { version = "0.4", default-features = false, features = ["array-sizes-33-128", "array-sizes-129-255"] }
 serde = { version = "1.0", optional = true }
-parity-scale-codec-derive = { path = "derive", version = "1.0", default-features = false, optional = true }
+parity-scale-codec-derive = { path = "derive", version = "^1.0.2", default-features = false, optional = true }
```

Comment:
Just curious: are there some differences between `^1.0.2` and `1.0.2`?
Cargo book [says](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio) the following:
> The string "0.1.12" is a semver version requirement. Since this string does not have any operators in it, it is interpreted the same way as if we had specified "^0.1.12", which is called a caret requirement.

Reply:
oh ok indeed it seems not